modules/msk/neoantigenutils/neoantigeninput/resources/usr/bin/generate_input.py

```diff
@@ -370,75 +413,100 @@ def find_most_similar_string(target, strings):
+                SV = True
+                WTid = (IDsplit[0]
+                        + IDsplit[1][0:8]
+                        + '_'
+                        + str(len(row_mut["peptide"]))
+                        + '_'
+                        + row_mut["MHC"].split("-")[1].replace(":", "").replace("*", "")
+                        + '_'
+                        + str(row_mut["pos"])
+                        )
+                noposID = (
+                    IDsplit[0]+"_"+IDsplit[1][0:8]
+                    + "_"
+                    + str(len(row_mut["peptide"]))
+                    + "_"
+                    + row_mut["MHC"].split("-")[1].replace(":", "").replace("*", "")
+                )
+                #this part makes the dict that matches this to the bedpe
+                bedpe_match_dict[row_mut["Identity"]]= IDsplit[0]+'_'+IDsplit[1][0:4]
+            else:
+                # first find match in WT
+                WTid = (
+                    row_mut["Identity"][:-2]
+                    + "_"
+                    + str(peplen)
+                    + "_"
+                    + row_mut["MHC"].split("-")[1].replace(":", "").replace("*", "")
+                    + "_"
+                    + str(row_mut["pos"])
+                )
+                noposID = (
+                    row_mut["Identity"][:-2]
+                    + "_"
+                    + str(peplen)
+                    + "_"
+                    + row_mut["MHC"].split("-")[1].replace(":", "").replace("*", "")
+                )
+            if WTid in WTdict and ('M' == IDsplit[1][0] and 'Sp' not in row_mut["Identity"]) or SV == False :
                 # match
                 matchfound = True
                 best_pepmatch = WTdict[WTid]["peptide"]
+                frameshift=False
 
             else:
-                if "-" in row_mut["Identity"] or "+" in row_mut["Identity"] and WTid in WTdict:
+                if "-" in row_mut["Identity"] or "+" in row_mut["Identity"] and WTid in WTdict or SV == False:
                     # Means there is a frame shift and we don't need to do a analysis of 5' end and 3' end as 3' end is no longer recognizeable/comparable to the WT sequence at all
                     # We can just move the windows along together. There will likely be little to no match with the WT peptides.
                     matchfound = True
                     best_pepmatch = WTdict[WTid]["peptide"]
-                    # print(mutation_dict[row_mut["Identity"]])
-
+                    frameshift=False
                 else:
+                    #Here we take care of frameshifted peptides
+                    frameshift=True
                     best_pepmatch,best_pepmatch2 , first_AA_same, first_AA_same_score, match_score = find_most_similar_string(row_mut["peptide"],list(WTdict[noposID]['peptides'].keys()))
-
-                    if best_pepmatch == row_mut["peptide"]:
+                    print((best_pepmatch,best_pepmatch2))
```

Comment:
Is this print statement supposed to be here or is it an artifact from debugging?